### PR TITLE
Add CONN_CAR handler to central server

### LINKED ISSUE #7: Add CONN_CAR handler

### PR DESCRIPTION
**Pull request by igorveselinovic**
Resolves #7

### CODE REVIEW

File: handlers.py [hunk 1]
```diff
@@ -121,6 +121,37 @@ def handle_register_car(server, body, source):
...
+
+    car_id = body['car_id']
+
+    if not car_id:
+        print('missing field: car_id')
+        server.send(Error.json(Error.BAD_REQ, 'missing field: car_id'), source)
+        return
+
+    dbconnect, cursor = _connect_to_db()
+    cursor.execute('select * from cars where (id=?)', (car_id,))
```
Comment (Thao-Tran) [inline handlers.py:135]: Is the comma after `car_id` meant to be there?

File: handlers.py [hunk 2]
```diff
@@ -121,6 +121,37 @@ def handle_register_car(server, body, source):
...
+    if entry is None:
+        msg = 'car does not exist'
+        print(msg)
+        server.send(Error.json(Error.BAD_REQ, msg), source)
+    elif entry[2] != request_ip:
+        msg = 'IP address does not match car ID'
+        print(msg)
+        server.send(Error.json(Error.BAD_REQ, msg), source)
+    else:
+        cursor.execute('update cars set isOn=1 where (id=?)', (car_id,))
```
Comment (Thao-Tran) [inline handlers.py:148]: Same with this comma.

File: handlers.py (hunk 1 above)
Comment (igorveselinovic) [reply handlers.py:135]: Yeah, it lets Python know that it's a tuple and not just an int.

Comment (Thao-Tran) [reply handlers.py:135]: Dope.

**Review by Thao-Tran — APPROVED** [review]

**Pull request merged**